src/components/Person/Person.jsx

```diff
@@ -1 +1,20 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
```

Comment:
use object destructing 
```suggestion
export const Person = ({ person }) => {
  const {...} = person;
```